app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/cloud/executor/AppEngineStandardRunTask.java

```diff
@@ -78,7 +78,7 @@ public AppEngineStandardRunTask(
   @Override
   public void execute(ProcessStartListener startListener) {
     // show a warning notification if the cloud sdk version is not supported
-    CloudSdkVersionNotifier.getInstance().notifyIfUnsupportedVersion();
+    CloudSdkVersionNotifier.getInstance().notifyIfVersionOutOfDate();
```

Comment:
`notifyIfVersionParseError()` is not applicable here?

Reply:
yeah. I think it is. I'll add it.